cmd/kube-controller-manager/controller-manager.go

```diff
@@ -102,6 +106,10 @@ func main() {
 	controllerManager := replicationControllerPkg.NewReplicationManager(kubeClient)
 	controllerManager.Run(10 * time.Second)
 
+	kubeletClient, err := client.NewKubeletClient(&kubeletConfig)
```

Comment:
The apiserver does:
client.BindKubeletClientConfigFlags(flag.CommandLine, &kubeletConfig)

Is that done somewhere here?